Modules/_elementtree.c

```diff
@@ -725,6 +727,24 @@ _elementtree_Element___copy___impl(ElementObject *self)
     return (PyObject*) element;
 }
 
+/*[clinic input]
+_elementtree.Element.copy
+
+[clinic start generated code]*/
+
+static PyObject *
+_elementtree_Element_copy_impl(ElementObject *self)
+/*[clinic end generated code: output=84660e8524276b22 input=1f8134305a7719a3]*/
+{
+    if (PyErr_WarnEx(PyExc_DeprecationWarning,
```

Comment:
I'm personally ok with deprecating this right away (lxml never provided such a `.copy()` method, AFAICT), but still, the usual process would be to issue a `PendingDeprecationWarning` first.